docs/core/preview3/tools/dotnet-clean.md

```diff
@@ -14,10 +14,8 @@ ms.assetid: eff65fa1-bab4-4421-8260-d0a284b690b2
 
 #dotnet-clean
 
-[!INCLUDE[preview-warning](../../../includes/warning.md)]
-
 ## Name 
-`dotnet-clean` -- Cleans the output of a project. 
+`dotnet-clean` - Cleans the output of a project. 
```

Comment:
nit: let's standardize on whether we'll show the command name with or without backticks since some commands are using and some are not. Probably without causes less confusion.